src/math/window.c

```diff
@@ -114,7 +115,7 @@ void win_povey_16b(int16_t win[], int length)
 		/* Calculate x^0.85 as exp(0.85 * log(x)) */
 		x2 = (int32_t)(ln_int32((uint32_t)x1) >> 1) - WIN_LOG_2POW31_Q26;
 		x3 = sat_int32(Q_MULTSR_32X32((int64_t)x2, WIN_085_Q31, 26, 31, 27)); /* Q5.27 */
-		x4 = exp_fixed(x3); /* Q5.27 -> Q12.20 */
+		x4 = sofm_exp_fixed(x3); /* Q5.27 -> Q12.20 */
```

Comment:
somehow, you can rename from the beginning or start from first patch, then this patch will be removed.